templates/components/badgeInstance.jsx

```diff
@@ -0,0 +1,24 @@
+var React = require('react');
+
+
+var BadgeInstance = React.createClass({
+  render: function() {
+    var badge = this.props.badge,
+        doiRe = /(10\.\d{3}\d+)\/(.*)\b/,
+        m = doiRe.exec(badge.evidenceUrl),
+        paperUrl = '/v/#/papers/' + m[1] + '/' + encodeURIComponent(m[2]) + '/badges';
+
+    return (
+      <div className="badge pure-u-1-4">
+          <div className="pure-u-1-2">
+            <img src={ badge.badge.imageUrl }/>
```

Comment:
this is a bit weird that you have to access two level of the same name here lol. `badge.badge` where in line 6 you already did `this.props.badge`...